@xen-orchestra/lite/src/stories/web-core/legend-title.story.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('title').type('string').required().preset('Title text').widget(),
+      prop('tooltip').type('string').widget(),
+      prop('icon').type('IconDefininition').preset(faInfoCircle).widget(),
+    ]"
+  >
+    <LegendTitle v-bind="properties">{{ settings.slot }}</LegendTitle>
+  </ComponentStory>
```

Comment:
- When documenting a icon prop, you can use the `iconProp()` helper. It will configure the type automatically and will add a selector with a set of predefined icons.
- You passed a value (`settings.slot`) to the default slot, but in your version there is no slot at all. Also, you used `settings.slot` from nowhere, as it does not exists.

```suggestion
  <ComponentStory
    v-slot="{ properties, settings }"
    :params="[
      iconProp(),
      prop('icon-tooltip').type('string').widget(),
      slot(),
      setting('defaultSlotContent').preset('Legend title').widget(text()),
    ]"
  >
    <LegendTitle v-bind="properties">{{ settings.defaultSlotContent }}</LegendTitle>
  </ComponentStory>
```